tests/test_w_dens.R

```diff
@@ -0,0 +1,58 @@
+here::here()
```

Comment:
There is a specific syntax for test to use in a package, using *testthat*. See other test files in this project for some examples. Here, the additional tests should include:

* testing that the warning is issued when it should, and that it is the exact message expected
* testing that the replacement of non-positive values is made when it should

If we go down the road of implementing `satitize_pmf` then ideally the test is in a file `test-satitize_pmf.R`.